src/main/java6/net/finmath/marketdata/model/curves/ForwardCurve.java

```diff
@@ -43,10 +45,7 @@
 	private InterpolationEntityForward	interpolationEntityForward = InterpolationEntityForward.FORWARD;
 
 	/**
-	 * Generate a forward curve using a given discount curve and payment offset. The forward F(t) of an index is such that
-	 * F(t) * D(t+p) equals the market price of the corresponding index fixed in t and paid in t+d, where t is the fixing time
-	 * of the index and t+p is the payment time of the index. F(t) is the corresponding forward and D is the associated discount
-	 * curve.

```

Comment:
I was wondering why this JavaDoc has to go, but now I see that you just moved it to the class documentation. Very good. +1